slimta/logging/__init__.py

```diff
@@ -128,7 +130,7 @@ def logline(log, type, typeid, operation, **data):
         log('{0}:{1}:{2}'.format(type, typeid, operation))
     else:
         data_str = ' '.join(['='.join((key, log_repr.repr(val)))
-                             for key, val in sorted(data.iteritems())])
+                             for key, val in sorted(six.iteritems(data))])
```

Comment:
I'd just change this to `data.items()`, which should work just fine on py3 and py2.